nodeup/pkg/model/kubelet.go

```diff
@@ -597,8 +597,10 @@ func (b *KubeletBuilder) buildKubeletConfigSpec() (*kops.KubeletConfigSpec, erro
 
 	// In certain configurations systemd-resolved will put the loopback address 127.0.0.53 as a nameserver into /etc/resolv.conf
 	// https://github.com/coredns/coredns/blob/master/plugin/loop/README.md#troubleshooting-loops-in-kubernetes-clusters
-	if b.Distribution == distros.DistributionBionic && c.ResolverConfig == nil {
-		c.ResolverConfig = s("/run/systemd/resolve/resolv.conf")
+	if c.ResolverConfig == nil {
+		if b.Distribution == distros.DistributionBionic || b.Distribution == distros.DistributionFocal {
```

Comment:
Out of interest, do you know that this was needed?  I was hoping they might have fixed it!

Reply:
This is a more general issue for distros that enable DNS cache via `systemd-resolved`, so not sure it can be fixed. Without this CoreDNS keeps complaining about a DNS loop:
https://github.com/coredns/coredns/blob/master/plugin/loop/README.md#troubleshooting-loops-in-kubernetes-clusters